website/docs/reference/resource-configs/schema.md

```diff
@@ -23,6 +23,11 @@ models:
 </File>
 
 This would result in the generated relations for these models being located in the  `marketing` schema, so the full relation names would be `analytics.marketing.model_name`. 
+
+[comment]: But this: `analytics.marketing.model_name` isn't true according to this: https://docs.getdbt.com/docs/build/custom-schemas#understanding-custom-schemas, because in fact, we're providing a custom schema, aren't we?
```

Comment:
@joellabes Can you verify this information?

Reply:
Yes this callout is correct - by default it will be {{ target.schema }}_{{ schema }} and we should link to https://docs.getdbt.com/docs/build/custom-schemas#understanding-custom-schemas to explain why

Reply:
thanks team, have made those change and folded them in!